### DISCUSSION TITLE
copilot indentation broken

### DISCUSSION THREAD

**Discussion by loiccoyle** [LazyVim/LazyVim#387] [Q&A] [answered]
With the `copilot` extra which was added recently, inserting a `copilot` snippet is kinda broken as all indentation is ignored, see this issue: https://github.com/zbirenbaum/copilot-cmp/issues/45

Apparently, the solution is to use `cmp.ConfirmBehavior.Replace` on `cmp.mapping.confirm`. As shown in this snippet https://github.com/zbirenbaum/copilot-cmp/issues/45#issuecomment-1454050516 

While this is clearly a `copilot-cmp` issue, maybe `LazyVim` should implement the workaround linked above in the `copilot` extra  for the time being?

**Comment by folke** [comment, marked as answer]
I just pushed an update to LazyVim for the copilot extra that does **Replace** but only for confirming a completion from the **copilot** source.

See https://github.com/LazyVim/LazyVim/commit/079d3967d03d47b95bdec10bb2621db1c4a07dce

**Reply by loiccoyle** [reply]
Perfect ! Thanks !

**Reply by folke** [reply]
I'm also going to see to add those cmp priorities. I've only been using copilot for a couple of days, but really liking it so far!

**Reply by loiccoyle** [reply]
yeah same, I saw you added it and have been using it since.
